### PR TITLE
Bugfix: Wrong count number in timeline list

### PR DESCRIPTION
**Pull request by berggren**
We were getting the wrong number (all shards together).

### CODE REVIEW

File: timesketch/lib/datastores/elastic.py [hunk 1]
```diff
@@ -594,18 +594,26 @@ def count(self, indices):
...
+        except NotFoundError:
             es_logger.error(
                 'Unable to count indexes (index not found)',
                 exc_info=True)
-            return 0
-        return result.get('count', 0)
+            es_stats = {}
+
+        doc_count_total = es_stats.get(
+            '_all', {}).get('primaries').get('docs', {}).get('count', 0)
```
Comment (kiddinn) [inline timesketch/lib/datastores/elastic.py:612]: maybe have {} as the default in other places too

.get('primaries', {}).get('docs....

and .get('primaries', {}).get('store', {}).get..... below

**Review by kiddinn — CHANGES_REQUESTED** [review]
CAn you start by fixing all the tests and the linter/

File: timesketch/lib/datastores/elastic.py (hunk 1 above)
Comment (berggren) [reply timesketch/lib/datastores/elastic.py]: good catch, fixed.

**Review by kiddinn — APPROVED** [review]
LGTM

**Pull request merged**